db/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express")
+const { getApi, getAllTopics } = require("./controllers/controllers")
+
+
+const app = express();
+app.use(express.json());
```

Comment:
Please remove your express.json until you need to use it 🙏 we shouldn't have code that is not needed for the ticket you are working on 🙂